addons/web_editor/static/lib/odoo-editor/src/commands/deleteBackward.js

```diff
@@ -203,7 +205,7 @@ HTMLElement.prototype.oDeleteBackward = function (offset, alreadyMoved = false,
     }
     if (cursorNode.nodeType !== Node.TEXT_NODE) {
         const { cType } = getState(cursorNode, cursorOffset, DIRECTIONS.LEFT);
-        if (cType & CTGROUPS.BLOCK && (!alreadyMoved || cType === CTYPES.BLOCK_OUTSIDE)) {
+        if (cType & CTGROUPS.BLOCK && !isFirstEl && (!alreadyMoved || cType === CTYPES.BLOCK_OUTSIDE)) {
```

Comment:
Could you please try the following code here and tell me if it works or not:
```js
if (
    !this.previousElementSibling && 
    ['BLOCKQUOTE','H1','H2','H3','PRE'].includes(this.nodeName)
) {
    cost p = document.createElement('p');
    p.replaceChildren(...this.childNodes);
    this.replaceWith(p);
} else if (cType & ... the current code in 15.0
```

Reply:
Hello, @dmo-odoo it's not working because by the time it reaches the block i guess element referred by "this" doesn't exists maybe due to moveNodes, maybe what we can do is cursorNode.appendChild(p) or inserBefore let me know thanks.

Reply:
I checked the branch locally and I think I have found a better place for this condition. I have changed the commit and force pushed the branch.